src/Spice86.Core/Emulator/InterruptHandlers/Input/Mouse/MouseInt33Handler.cs

```diff
@@ -564,8 +564,18 @@ private void FillDispatchTable() {
         AddAction(0x1A, SetMouseSensitivity);
         AddAction(0x1B, GetMouseSensitivity);
         AddAction(0x1C, SetInterruptRate);
+        AddAction(0x21, Reset);
         AddAction(0x24, GetSoftwareVersionAndMouseType);
     }
+    
+    private void Reset() {
```

Comment:
This hides unimplemented interrupts with logs, I'd rather follow @kevinferrare design and remove this (and changes in the calling code) to let the exception crash the emulator on unimplemented behavior, or implement the operation.